src/main/java/fr/foodlimit/api/shared/models/User.java

```diff
@@ -23,11 +24,21 @@
 
   private String password;
 
+  @JsonIgnore
   @OneToMany(mappedBy = "user", cascade = CascadeType.ALL)
   private List<Food> foods;
 
   public void encodePassword(PasswordEncoder passwordEncoder) {
     this.password = passwordEncoder.encode(this.password);
   }
 
+  @Override
+  public String toString() {
+    return "User{" +
+      "username='" + username + '\'' +
+      ", name='" + name + '\'' +
+      ", email='" + email + '\'' +
+      ", password='" + password + '\'' +
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') 'password' detected in this expression, review this potentially hardcoded credential. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2068)